ttkwidgets/frames/vnotebook.py

```diff
@@ -0,0 +1,312 @@
+            args = ("<Control-Tab>", self._switch_tab,)
+        else:
+            func = "unbind"
+            args = ("<Control-Tab>",)
+        for widget in (self, self._buttons_frame, self._separator) + tuple(self._tab_frames.values()):
+            getattr(widget, func)(*args)
+        return enable
+
+    def disable_traversal(self):
+        """Alias of :obj:`VNotebook.enable_traversal(enable=False)`."""
+        return self.enable_traversal(enable=False)
+
+    def forget(self, child):
+        """Remove a child by widget or tab_id."""
+        tab_id = self.get_id_for_tab(child)
+        self._tab_buttons[tab_id].destroy()
+        del self._tab_buttons[tab_id]
+        del self._tab_frames[tab_id]
+        self._tab_ids.remove(tab_id)
+        self._grid_tabs()
+
+    def hide(self, child, hide=True):
+        """Hide or unhide a Tab."""
+        tab_id = self.get_id_for_tab(child)
+        self._hidden[tab_id] = hide
+        if tab_id == self.active and len(self._tab_ids) != 1:
+            self.activate(self._tab_ids[0])
+        self.grid_widgets()
+
+    def show(self, child):
+        """Alias for :obj:`VNotebook.hide(hide=False)`"""
+        return self.hide(child, hide=False)
+
+    def index(self, child):
+        """Return zero-indexed index value of a child or tab_id."""
+        return self._tab_ids.index(self.get_id_for_tab(child))
+
+    def tab(self, tab_id, option=None, **kwargs):
+        """
+        Configure a tab with options given in kwargs.
+
+        :param tab_id: Non-optional tab ID of the tab to be configured
+        :param option: If not None, function returns value for option
+            key given in this argument
+        """
+        if option is not None:
+            return self._tab_buttons[tab_id].cget(option)
+        # Argument processing
+        self._frame_padding[tab_id] = \
+            (kwargs.pop("padding", self._padding), kwargs.pop("sticky", tk.N))
+        kwargs["command"] = lambda tab_id=tab_id: self.activate(tab_id)
+        kwargs["style"] = "Toolbutton"
+        # Configure Buttons
+        self._tab_buttons[tab_id].configure(**kwargs)
+        self._grid_tabs()
+
+    def tab_configure(self, tab_id, **kwargs):
+        """Configure alias for :meth:`VNotebook.tab`"""
+        return self.tab(tab_id, **kwargs)
+
+    def tab_cget(self, tab_id, key):
+        """cget alias for :meth:`VNotebook.tab`"""
+        return self.tab(tab_id, option=key)
+
+    @property
+    def tabs(self):
+        """Return list of tab IDs."""
+        return self._tab_ids.copy()
+
+    def config(self, **kwargs):
+        """Alias for :meth:`VNotebook.configure`"""
+        return self.configure(**kwargs)
+
+    def configure(self, **kwargs):
+        """Change settings for the widget."""
+        for option in self.options:
+            attr = "_{}".format(option)
+            setattr(self, attr, kwargs.pop(option, getattr(self, attr)))
+        return ttk.Frame.configure(**kwargs)
```

Comment:
for the `self` missing, shouldn't we just use `super()` instead of calling `ttk.Frame` directly ? It's much cleaner after all.

Reply:
Yes we can use `super()` though I have not taken the habit to use it yet, but this does not solve the issue of the behavior of the function. I don't think that many people use `.configure("<key>")`  since the output is not really usable in python (e.g. ` ('width', 'width', 'Width', 0, 0)`). So maybe we can return nothing and just use the method for its main purpose which is to change the options of the widget.

Reply:
I removed the call to ttk.Frame, in favor of a super() call. We could just return whatever ttk.Frame.configure returns, that way it stays consistent with the rest of the tkinter framework.